inference-engine/include/ie_api.h

```diff
@@ -9,14 +9,18 @@
  */
 #pragma once
 
-#if defined(USE_STATIC_IE) || (defined(__GNUC__) && (__GNUC__ < 4))
+#ifdef __GNUC__
+# define INFERENCE_ENGINE_CDECL __attribute__((cdecl))
+# else
+# define INFERENCE_ENGINE_CDECL __cdecl
+#endif
+
+#if (!defined(__GNUC__) && defined(USE_STATIC_IE)) || (defined(__GNUC__) && (__GNUC__ < 4))
```

Comment:
why do we need `!defined(__GNUC__)` here? Does not look consistent with other similar places.

Reply:
Otherwise i have to add a #ifdef USE_STATIC_LIB inside the #if defined(_WIN32)